WordPress/Classes/ViewRelated/Post/PostAuthorSelectorViewController.swift

```diff
@@ -0,0 +1,82 @@
+import UIKit
+
+@objc class PostAuthorSelectorViewController: SettingsSelectionViewController {
+    /// The post to change the author.
+    private var post: AbstractPost!
+
+    /// A completion block that is called after the user selects an option.
+    @objc var completion: (() -> Void)?
```

Comment:
This might be able to go in the initializer along with `post`, but that might make things more complicated at the call site where `PostAuthorSelectorViewController` is initialized, so I'm happy to leave as-is.

Reply:
I generally like to inject the dependency at the call site when possible as well, but I'm good either way. I'm fine with it as well.

Reply:
Sounds good!